rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -338,6 +338,7 @@ public Page<JobInfo> getJobs(int index, int range, JobFilterCriteria criteria,
                                                                                                            criteria.isChildJobs(),
                                                                                                            criteria.getJobName(),
                                                                                                            criteria.getProjectName(),
+                                                                                                           criteria.getBucketName(),
```

Comment:
in the method getJobInfo(..) of this class you should add

jobInfoImpl.setBucketName(jobInfoData.getBucketName());

Please do so for other recently added attributes if not already.